cmd/otelcol/config/collector/agent_config.yaml

```diff
@@ -4,6 +4,8 @@
 # environment variables are required to be manually defined or configured below:
 # - SPLUNK_ACCESS_TOKEN: The Splunk access token to authenticate requests
 # - SPLUNK_API_URL: The Splunk API URL, e.g. https://api.us0.signalfx.com
+# - SPLUNK_BUNDLE_DIR: The path to the Smart Agent bundle, e.g. /usr/lib/splunk-otel-collector
```

Comment:
Does `/usr/lib/splunk-otel-collector/agent-bundle` make more sense?  We will likely have more artifacts over time and having everything all together in one directory seems undesirable.